packages/fxa-settings/src/pages/InlineTotpSetup/index.tsx

```diff
@@ -164,29 +165,13 @@ export const InlineTotpSetup = ({
             />
           )}
           <section>
-            <form noValidate>
```

Comment:
This was causing errors in our tests:

![image](https://user-images.githubusercontent.com/13018240/216388231-12cd917f-da38-4286-ad94-faeb9e71dd70.png)